web/themes/custom/pineapple/components/_comments.twig

```diff
@@ -0,0 +1,114 @@
+ *     eg. by-node-author.
+ *   - preview: When previewing a new or edited comment.
+ *   The following applies only to viewers who are registered users:
+ *   - unpublished: An unpublished comment visible only to administrators.
+ * - title_prefix: Additional output populated by modules, intended to be
+ *   displayed in front of the main title tag that appears in the template.
+ * - title_suffix: Additional output populated by modules, intended to be
+ *   displayed after the main title tag that appears in the template.
+ * - content_attributes: List of classes for the styling of the comment content.
+ * - title_attributes: Same as attributes, except applied to the main title
+ *   tag that appears in the template.
+ * - threaded: A flag indicating whether the comments are threaded or not.
+ *
+ * These variables are provided to give context about the parent comment (if
+ * any):
+ * - parent_comment: Full parent comment entity (if any).
+ * - parent_author: Equivalent to author for the parent comment.
+ * - parent_created: Equivalent to created for the parent comment.
+ * - parent_changed: Equivalent to changed for the parent comment.
+ * - parent_title: Equivalent to title for the parent comment.
+ * - parent_permalink: Equivalent to permalink for the parent comment.
+ * - parent: A text string of parent comment submission information created from
+ *   'parent_author' and 'parent_created' during template_preprocess_comment().
+ *   This information is presented to help screen readers follow lengthy
+ *   discussion threads. You can hide this from sighted users using the class
+ *   visually-hidden.
+ *
+ * These two variables are provided for context:
+ * - comment: Full comment object.
+ * - entity: Entity the comments are attached to.
+ *
+ * @see template_preprocess_comment()
+ */
+#}
+{% if threaded %}
+  {{ attach_library('classy/indented') }}
+{% endif %}
+{%
+  set classes = [
+  'comment',
+  'js-comment',
+  status != 'published' ? status,
+  comment.owner.anonymous ? 'by-anonymous',
+  author_id and author_id == commented_entity.getOwnerId() ? 'by-' ~ commented_entity.getEntityTypeId() ~ '-author',
+  'u-o-4',
+]
+%}
+<article{{ attributes.addClass(classes) }}>
+  {#
+    Hide the "new" indicator by default, let a piece of JavaScript ask the
+    server which comments are new for the user. Rendering the final "new"
+    indicator here would break the render cache.
+  #}
+  <mark class="hidden new-label" data-comment-timestamp="{{ new_indicator_timestamp }}"></mark>
+
+  {% set submitted_attributes = submitted_attributes|default(create_attribute()).addClass([
+    'comment-submitted',
+    author_id == 1 ? 'admin-comment',
+  ]) %}
+
+  <footer class="comment__meta">
+    {{ user_picture }}
+
+    {% set author_name = comment.getAuthorName() %}
+    <p{{ submitted_attributes }}>{{ author_name }} says</p>
+    <a href="{{ comment.url() }}">{{ created }}</a>
```

Comment:
Do these need to be on different lines? it makes the thing look clunky unnecessarily In my opinion and it doesn't read well... [Name of author] says [time of post] [actual post], this should probably be reconfigured

Reply:
I disagree. It matches the original designs and my opinion is that it looks fine and is clear enough. I don't want to reconfigure this.